src/Tickets/Flexible_Tickets/Series_Passes.php

```diff
@@ -132,6 +144,8 @@ public function is_active(): bool {
 	protected function do_register(): void {
 		$this->container->singleton( Series_Passes\Repository::class, Series_Passes\Repository::class );
 		$this->container->singleton( Series_Passes\Metadata::class, Series_Passes\Metadata::class );
+		// Register reporting functions.
+		$this->reports->register_hooks();
```

Comment:
See my other note: **only controllers** should hook actions and filters.

Reply:
Looks like this is set as tolerable and is working as expected to avoid conflict: https://github.com/the-events-calendar/event-tickets-plus/blob/c3d06592c99c9767cb5f12346262f310cbc8ea09/tests/wpunit/Tribe/Tickets_Plus/Partials/V2/Modal/Attendee_Registration_Test.php#L48